README.md

```diff
@@ -637,6 +637,14 @@ $ lerna link
 
 Symlink together all Lerna `packages` that are dependencies of each other in the current Lerna repo.
 
+#### --ignore-semver
+
+```sh
+$ lerna link --ignore-semver=<boolean>
```

Comment:
Can we rename this `--force-local`, please? Even though it _does_ ignore semver, in practice, I don't want to implicitly condone ignoring semver. It's pure semantics, yes, but I think `--force-local` better describes what is actually being accomplished, rather that how it is implemented.

Also, for boolean CLI options, it's not necessary to pass a value:
```sh
$ lerna link --force-local
```
(omitting the flag entirely is the equivalent of `false`, the default)